coa/pkg/apis/v1alpha2/providers/states/redisstate/redisstate.go

```diff
@@ -29,7 +29,7 @@ var rLog = logger.NewLogger("coa.runtime")
 
 const (
 	entryCountPerList = 100
-	separator         = ":"
+	separator         = "*"
```

Comment:
Can be constant value. Or this is internal used only?

Reply:
It's used to solve this issue: https://github.com/eclipse-symphony/symphony/issues/389

Reply:
This is only used in Redis state provider so I think define in this module is enough.